### PR TITLE
Ignore German locale test if locale not installed

### PR DESCRIPTION
**Pull request by jart**

### CODE REVIEW

File: tensorboard/loader_test.py [hunk 1]
```diff
@@ -228,7 +228,10 @@ def testBigNumberInAmerica_showsCommas(self):
     self.assertIn(u' 1,024B/s ', self.bars[0])
 
   def testBigNumberInGermany_showsCommas(self):
-    locale.setlocale(locale.LC_ALL, 'de_DE.utf8')
+    try:
+      locale.setlocale(locale.LC_ALL, 'de_DE.utf8')
+    except Exception:
+      return  # internal CI doesn't have this locale for some reason
```
Comment (wchargin) [inline tensorboard/loader_test.py]: Could we substitute a different locale instead? What locales does the internal CI support?

Comment (chihuahua) [inline tensorboard/loader_test.py]: Maybe `tf.logging.info` this case.

Comment (chihuahua) [inline tensorboard/loader_test.py]: That seems to work too if feasible.

Comment (wchargin) [inline tensorboard/loader_test.py]: Why?

**Comment by wchargin** [comment]
Soft-request-changes to prefer keeping this test if you can. "Disable test because build system didn't do what we thought it did" isn't a great commit summary.

File: tensorboard/loader_test.py (hunk 1 above)
Comment (chihuahua) [inline tensorboard/loader_test.py]: Mainly since it deviates from the main flow, ie the test passes but trivially.

Comment (wchargin) [inline tensorboard/loader_test.py]: Sure, but why log it? No one would ever read that log.

Comment (wchargin) [inline tensorboard/loader_test.py]: If we can't find a way around keeping the test, prefer `self.skipTest("Google-internal CI doesn't support the de_DE.utf8 locale")`.

Comment (chihuahua) [inline tensorboard/loader_test.py]: Indeed, folks rarely read logs if nothing's awry. They might occasionally be useful though, ie I wonder why this test passes no matter how I change something. Maybe a warning.

Comment (chihuahua) [inline tensorboard/loader_test.py]: That sounds rather ideal. If we can skip an individual test.

Comment (jart) [inline tensorboard/loader_test.py]: It won't even show the log if the tests pass. Not much sense in logging. I could always just delete this test.

Comment (jart) [inline tensorboard/loader_test.py]: There's probably some thing we need to link against, but I'm having trouble finding it. I don't care about this test enough to spend too much time thinking about it. It works on Travis. I can also delete it.

Comment (wchargin) [inline tensorboard/loader_test.py]: Okay—if you don't care about the test, that's fine with me. Feel free to `skipTest` it or delete it.

Comment (jart) [inline tensorboard/loader_test.py]: Done.

**Comment by jart** [comment]
PTAL

File: tensorboard/loader_test.py [hunk 2]
```diff
@@ -228,7 +228,10 @@ def testBigNumberInAmerica_showsCommas(self):
     self.assertIn(u' 1,024B/s ', self.bars[0])
 
   def testBigNumberInGermany_showsCommas(self):
-    locale.setlocale(locale.LC_ALL, 'de_DE.utf8')
+    try:
+      locale.setlocale(locale.LC_ALL, 'de_DE.utf8')
+    except Exception:
+      raise self.skipTest('Environment does not support de_DE.utf8 locale')
```
Comment (wchargin) [inline tensorboard/loader_test.py]: (This…will work, but it's easier just to write `self.skipTest`, which raises the exception itself. :-) )

File: tensorboard/loader_test.py [hunk 3]
```diff
@@ -228,7 +228,10 @@ def testBigNumberInAmerica_showsCommas(self):
     self.assertIn(u' 1,024B/s ', self.bars[0])
 
   def testBigNumberInGermany_showsCommas(self):
-    locale.setlocale(locale.LC_ALL, 'de_DE.utf8')
+    try:
+      locale.setlocale(locale.LC_ALL, 'de_DE.utf8')
+    except Exception:
```
Comment (wchargin) [inline tensorboard/loader_test.py]: Wait, why did this raise a lint error [here](https://travis-ci.org/tensorflow/tensorboard/jobs/258297196) but not [here](https://travis-ci.org/tensorflow/tensorboard/jobs/258355256)? Both have this line.

**Pull request merged**